Orange/classification/rules.py

```diff
@@ -1,3 +1,12 @@
+"""
+Induction of rules works by finding a rule that covers some learning instances,
+removing these instances, and repeating this until all instances are covered.
+The rules are scored by heuristics such as impurity of class distribution of
+the covered instances. The module includes the common rule-learning algorithms,
```

Comment:
I understand the first one, but not the second. There are some common rule-learning algorithms, namely two variants of CN2. Wouldn't these then be *the* common rule-learning algorithms?